TRUCSBot/Commands/GameNightSuggestionCommands.cs

```diff
@@ -36,17 +35,31 @@ public async Task AddToBoard(CommandContext ctx, [Description("The title of the
 
             try
             {
-                var games = await igdb.QueryAsync<Game>(IGDBClient.Endpoints.Games, $"search \"{title}\"; fields id,name,cover.*,involved_companies.company.name,platforms.name,summary,url;");
-                var game = games.OrderBy(x => x.Name).FirstOrDefault();
-                if (game != null)
+                var games = await _igdb.QueryAsync<Game>(IGDBClient.Endpoints.Games, $"search \"{title.Replace("\"", "\\\"")}\"; fields id,name,cover.*,involved_companies.company.name,platforms.name,summary,url,aggregated_rating;");
+                if (games.Length > 0)
                 {
+                    Array.Sort(games, (a, b) =>
```

Comment:
I believe this could be simplified to something like

```csharp
games.OrderBy(x -> LevenshteinDistance(title, x.name)).ThenBy(x => x.AggregatedRating);
```

(you'd have to check whether to use the descending versions though)